pandas/core/arrays/categorical.py

```diff
@@ -2864,6 +2864,11 @@ class CategoricalAccessor(PandasDelegate, PandasObject, NoNewAttributesMixin):
     ----------
     data : Series or CategoricalIndex
 
+    See Also
+    --------
+    Series.dt : Accessor object for datetimelike properties of the Series values.
+    Series.sparse : Accessor for SparseSparse from other sparse matrix data types.
```

Comment:
`SparseSparse`?

Reply:
I copied the description from [pandas.Series.sparse](https://pandas.pydata.org/docs/reference/api/pandas.Series.sparse.html). If this is wrong, I can fix it there too.

Reply:
I would suggest removing `SparseSparse from other`